README.md

```diff
@@ -28,47 +28,20 @@
 
 
 ## Getting Started
```

Comment:
- [ ] I was unable to start up your project because you were missing `uuid` and `redux` npm packages (see image). Kindly run the commands in the error messages to fix this

  ![run-npm](https://user-images.githubusercontent.com/36542986/173550049-a0108cec-7766-45ed-9f7c-6379695f8013.jpg)